homeassistant/components/media_player/__init__.py

```diff
@@ -1057,6 +1061,25 @@ def __init__(
         self.children = children
         self.children_media_class = children_media_class
         self.thumbnail = thumbnail
+        self.sort()
+
+    @property
+    def is_sortable(self):
+        """Define the browsable media file as sortable when its media_content_type is movies or TV Shows."""
+        return (
+            self.media_content_type == MEDIA_TYPE_MOVIE
+            or self.media_content_type == MEDIA_TYPE_TVSHOW
+        )
+
+    @property
+    def sortable_title(self):
```

Comment:
This is logic that should be added to each integration as they know best how to sort it. We can't do this generic.

Reply:
When you say each integration, do you mean each individual one e.g. Kodi, Plex, etc?

Reply:
Yeah

Reply:
Done, I think haha
I set the sortable_title property with a default value of None unless the calling code passes it in to the constructor. The `sort()` call is now handled by Kodi's browse_media.py